okhttp-sse/build.gradle

```diff
@@ -16,3 +16,18 @@ dependencies {
   testImplementation deps.assertj
   testCompileOnly deps.jsr305
 }
+
+apply plugin: 'me.champeau.gradle.japicmp'
```

Comment:
Move to line 2

Reply:
Fix coming

Reply:
https://github.com/square/okhttp/pull/4898